digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGenerator.scala

```diff
@@ -712,6 +717,29 @@ case class TrafficSignProhibitionGenerator(roadLinkServiceImpl: RoadLinkService)
     if(value.nonEmpty) Some(Prohibitions(value)) else None
   }
 
+  override def createSegmentPieces(actualRoadLink: RoadLink, allRoadLinks: Seq[RoadLink], sign: PersistedTrafficSign, signs: Seq[PersistedTrafficSign], pointOfInterest: (Option[Point], Option[Point], Option[Int]), result: Seq[TrafficSignToLinear]): Set[TrafficSignToLinear] = {
+    if (debbuger) println("createSegmentPieces")
```

Comment:
vois vaihtaa logger

Reply:
varmaan pitää tehdä tiketti vaihtaa nää kaikki println täälä logger tai jos ei ole liian iso homma tehdä se nyt

Reply:
näitä println on jonkin verran

Reply:
varmaan jäänteitä kun ajettu eräajona